bundles/AdminBundle/Resources/public/js/pimcore/document/document.js

```diff
@@ -635,5 +634,14 @@ pimcore.document.document = Class.create(pimcore.element.abstract, {
                 this.data.key = rdata.key;
             }.bind(this)
         });
+    },
+
+    getIconClass: function () {
+        var iconClass;
+        if (this.data.iconCls) {
+            iconClass = this.data.iconCls;
+        } else if (this.data.icon) {
+            iconClass = pimcore.helpers.getClassForIcon(this.data.icon);
+        }
```

Comment:
No return value?

Reply:
We should move `getIconClass()` to `pimcore.element.abstract` then we can reuse it for assets.